setup.py

```diff
@@ -34,11 +34,14 @@ def find_stub_files(name):
     install_requires=dependencies,
     packages=["rest_framework-stubs", *find_packages(exclude=["scripts"])],
     package_data={"rest_framework-stubs": find_stub_files("rest_framework-stubs")},
+    python_requires=">=3.6",
     classifiers=[
-        "Development Status :: 3 - Alpha",
         "License :: OSI Approved :: MIT License",
+        "Operating System :: OS Independent",
         "Programming Language :: Python :: 3.6",
         "Programming Language :: Python :: 3.7",
         "Programming Language :: Python :: 3.8",
+        "Programming Language :: Python :: 3.9",
```

Comment:
We don't actually run tests on `3.9`, don't we? https://github.com/typeddjango/djangorestframework-stubs/blob/master/.travis.yml#L10

Btw, yesterday I have moved from travis to github actions: https://github.com/typeddjango/pytest-mypy-plugins/tree/master/.github/workflows

Maybe we should also?

Reply:
TBH we only need to run tests on 3.6 IMO, if it passes.it will pass for later versions because the changes are only typings.

As for github actions - sure, but maybe make this in a separate PR after this one?

Reply:
Ok!

Reply:
> TBH we only need to run tests on 3.6 IMO

It is better to run tests on all python versions we officially support. Better safe than sorry.